### DISCUSSION TITLE
How the js and css build command execute when the container is running?

### DISCUSSION THREAD

**Discussion by cris-m** [nickjj/docker-flask-example#11] [Q&A] [answered]
1. I notice that if  `watch` option is`true`  inside `esbuild.config.js` file, the `assets` container will hang during build. 
2. Regarding the `yarn:build:css` and `yarn:build:s` command. They won't execute because the `js` and `cs` container will all be terminate after building. 

How to achieve `watch` on `js` and `css` container in development?

**Comment by nickjj** [comment, marked as answer]
Hi,

The project is already set up to watch in development. The esbuild `watch` option keeps the container running and watches files for changes, then it immediately rebuilds the new JS on change. Tailwind works the same with the `--watch` flag for CSS.

You just need to run the app in development mode which it does by default in the .env file with:

```sh
export FLASK_DEBUG=true
export NODE_ENV=development
```
